### PR TITLE
Update `free-courses-hi.md` with Shell, Flutter, Bootstrap, jQuery, Node.js lists

### PR DESCRIPTION
**Pull request by GH05T-HUNTER5**
Hindi New Updates 
Bash and Shell , Flutter , Bootstrap, jQuery, Node.js

### CODE REVIEW

**Comment by GH05T-HUNTER5** [comment]
I don't know why this error

**Comment by davorpa** [comment]
> I don't know why this error

I also don't know why you submit incremental PRs masking commits even when **we aim you stop doing this in other PRs**

About push errors... see [the details of our GitHub linter task](https://github.com/EbookFoundation/free-programming-books/runs/7842640946?check_suite_focus=true)

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -42,6 +47,13 @@
 * [Complete Android Development Course in Hindi](https://www.youtube.com/playlist?list=PLUhfM8afLE_Ok-0Lx2v9hfrmbxi3GgsX1) - Neat Roots
 
 
+### Bash and Shell
+
+* [shell scripting complete tutorial in hindi](https://youtube.com/playlist?list=PL9A0tISr5Ow5nZSY8-ICNAyXHJKesl_XL) - Cybersploit
+* [BASH SHELL SCRIPTING (NOOB)](https://youtube.com/playlist?list=PLzOLSdbK1deNuVMOw0EkKGSsES-rPeONe) - Noob Coders
```
Comment (eshellman) [inline courses/free-courses-hi.md:53]: ALL CAPS titles not allowed.

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -42,6 +47,13 @@
...
+* [BASH BASIC COMMANDS](https://youtube.com/playlist?list=PLzOLSdbK1deOKmOiiv-o4wn7xUj6ZYzrM) - Noob Coders
```
Comment (eshellman) [inline courses/free-courses-hi.md:54]: ALL CAPS titles not allowed.

**Comment by eshellman** [comment]
> 
> I also don't know why you submit incremental PRs masking commits even when **we aim you stop doing this in other PRs**

Do not close a PR with comments and then create a new PR to try to fix the PR. It's disrespectful to the reviewers.

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -85,6 +97,12 @@
 * [DevOps Tutorials in Hindi/Urdu \| Devops सीखने का सबसे आसान तरीका \| Complete Devops including Git,Jenkins,Maven,Chef,docker,Ansible and Kubernetes](https://www.youtube.com/playlist?list=PLBGx66SQNZ8aPsFDwb79JrS2KQBTIZo10) - Bhupinder Rajput, Technical Guftgu
 
 
+### Flutter
+
+* [Complete Flutter Tutorial In Hindi By Desi Programmer](https://youtube.com/playlist?list=PLlFwzkUNmr94BF0KH7BYPL7DsZjhJRdTm) - Desi Programmer
+* [Flutter Tutorial For Beginners in Hindi](https://youtube.com/playlist?list=PLMkkZSS5OjPIwDyHHKVex6zr008U1-sWM) - Geeks Rank
```
Comment (davorpa) [inline courses/free-courses-hi.md:103]: Patch to recover a diff found with #7005 at commit 9fadbf98af17205729f3e562a4fcbd610b83bdf0 but alphabetizing Flutter list: https://github.com/EbookFoundation/free-programming-books/pull/7005/commits/9fadbf98af17205729f3e562a4fcbd610b83bdf0#diff-0bed9be5a9c29db386438a12b2e1abaac6945680b2e2ab9d07c0b58063e315adR93
```suggestion
* [Flutter 3 tutorial in Hindi](https://youtube.com/playlist?list=PLB97yPrFwo5g-XcPlfSXSOeeby23jVAcp) - CODERS NEVER QUIT
* [Flutter Tutorial For Beginners in Hindi](https://youtube.com/playlist?list=PLMkkZSS5OjPIwDyHHKVex6zr008U1-sWM) - Geeks Rank
```

File: courses/free-courses-hi.md [hunk 4]
```diff
@@ -99,6 +118,13 @@
 * [Web Development Tutorials for Beginners in Hindi: HTML, CSS, JavaScript and more](https://www.youtube.com/playlist?list=PLu0W_9lII9agiCUZYRsvtGTXdxkzPyItg) - CodeWithHarry
 
 
+#### Bootstrap
+
+* [Bootstrap 3 Tutorial](https://youtube.com/playlist?list=PLjVLYmrlmjGciJ5_Ze6jDIfJpEeMYtsSe) - WsCube Tech
+* [Bootstrap Tutorial In Hindi](https://youtube.com/playlist?list=PLdPwRNmUlk0k91-qAXTHFqMScNEuo8E5d) - CODE4EDUCATION
```
Comment (davorpa) [inline courses/free-courses-hi.md:124]: Patch to recover 3 diffs with #7005 committed at 9fadbf98af17205729f3e562a4fcbd610b83bdf0 deleted at 8b60e6f190880a378a43ad54b0ef3b3ec43f550e but alphabetizing this section: 
https://github.com/EbookFoundation/free-programming-books/pull/7005/commits/9fadbf98af17205729f3e562a4fcbd610b83bdf0#diff-0bed9be5a9c29db386438a12b2e1abaac6945680b2e2ab9d07c0b58063e315adR110-R114

```suggestion
* [Bootstrap Tutorial for Beginners(Hindi)](https://youtube.com/playlist?list=PLgPJX9sVy92wc38jA6JtvkA4l1xmJcKKH) - CS Geeks
* [Bootstrap Tutorial In Hindi](https://youtube.com/playlist?list=PLdPwRNmUlk0k91-qAXTHFqMScNEuo8E5d) - CODE4EDUCATION
* [Bootstrap Tutorial In Hindi](https://youtu.be/vpAJ0s5S2t0) - CodeWithHarry
* [Bootstrap Tutorial in Hindi With 2 Projects for Beginners \| Complete Bootstrap 5 Tutorial in Hindi](https://youtu.be/QE5oQh63gGE) - Tech Gun
```

File: courses/free-courses-hi.md [hunk 5]
```diff
@@ -99,6 +118,16 @@
 * [Web Development Tutorials for Beginners in Hindi: HTML, CSS, JavaScript and more](https://www.youtube.com/playlist?list=PLu0W_9lII9agiCUZYRsvtGTXdxkzPyItg) - CodeWithHarry
 
 
+#### Bootstrap
+
+* [Bootstrap 3 Tutorial](https://youtube.com/playlist?list=PLjVLYmrlmjGciJ5_Ze6jDIfJpEeMYtsSe) - WsCube Tech
+* [Bootstrap Tutorial for Beginners(Hindi)](https://youtube.com/playlist?list=PLgPJX9sVy92wc38jA6JtvkA4l1xmJcKKH) - CS Geeks
+* [Bootstrap Tutorial In Hindi](https://youtube.com/playlist?list=PLdPwRNmUlk0k91-qAXTHFqMScNEuo8E5d) - CODE4EDUCATION
+* [Bootstrap Tutorial In Hindi](https://youtu.be/vpAJ0s5S2t0) - CodeWithHarry
```
Comment (davorpa) [inline courses/free-courses-hi.md:126]: Expand YouTube shortened url.

Resolves suggestion made at https://github.com/EbookFoundation/free-programming-books/pull/6999#discussion_r945164659

```suggestion
* [Bootstrap Tutorial In Hindi](https://youtube.com/watch?v=vpAJ0s5S2t0) - CodeWithHarry
```

File: courses/free-courses-hi.md [hunk 6]
```diff
@@ -99,6 +118,16 @@
...
+* [Bootstrap Tutorial in Hindi With 2 Projects for Beginners \| Complete Bootstrap 5 Tutorial in Hindi](https://youtu.be/QE5oQh63gGE) - Tech Gun
```
Comment (davorpa) [inline courses/free-courses-hi.md:127]: Expand YouTube shortened url.

Resolves suggestion made at https://github.com/EbookFoundation/free-programming-books/pull/6999#discussion_r945164659

```suggestion
* [Bootstrap Tutorial in Hindi With 2 Projects for Beginners \| Complete Bootstrap 5 Tutorial in Hindi](https://youtube.com/watch?v=QE5oQh63gGE) - Tech Gun
```

Comment (davorpa) [reply courses/free-courses-hi.md]: Fixed by commit 8b7ae419ae60a37a755c9a91ed406fc7c54b7030

Comment (davorpa) [reply courses/free-courses-hi.md]: Fixed by commit 8b7ae419ae60a37a755c9a91ed406fc7c54b7030

File: courses/free-courses-hi.md (hunk 5 above)
Comment (davorpa) [reply courses/free-courses-hi.md]: Fixed by commit 8b7ae419ae60a37a755c9a91ed406fc7c54b7030

**Comment by davorpa** [comment]
According to last run task by GitHub Actions... the current linter faults are:

```
courses/free-courses-hi.md
 52:1-54:108  warning  Alphabetical ordering: swap l.53 and l.52    alphabetize-lists  remark-lint
 52:1-54:108  warning  Alphabetical ordering: swap l.54 and l.53    alphabetize-lists  remark-lint
 123:1-128:130  warning  Alphabetical ordering: swap l.128 and l.127  alphabetize-lists  remark-lint
```

**Comment by GH05T-HUNTER5** [comment]
> According to [last run task by GitHub Actions](https://github.com/EbookFoundation/free-programming-books/runs/7868953747?check_suite_focus=true) ... the current linter faults are:
> 
> ```
> courses/free-courses-hi.md
>  52:1-54:108  warning  Alphabetical ordering: swap l.53 and l.52    alphabetize-lists  remark-lint
>  52:1-54:108  warning  Alphabetical ordering: swap l.54 and l.53    alphabetize-lists  remark-lint
>  123:1-128:130  warning  Alphabetical ordering: swap l.128 and l.127  alphabetize-lists  remark-lint
> ```

@davorpa @eshellman 
Let me close these

File: courses/free-courses-hi.md (hunk 2 above)
Comment (davorpa) [reply courses/free-courses-hi.md:54]: As suggested by Eric...

```suggestion
* [Bash Basic Commands](https://youtube.com/playlist?list=PLzOLSdbK1deOKmOiiv-o4wn7xUj6ZYzrM) - Noob Coders
```

File: courses/free-courses-hi.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-hi.md:53]: As suggested by Eric...

```suggestion
* [Bash Shell Scripting (NOOB)](https://youtube.com/playlist?list=PLzOLSdbK1deNuVMOw0EkKGSsES-rPeONe) - Noob Coders
```

Comment (davorpa) [reply courses/free-courses-hi.md]: Fixed in commit 4b4b8b1bfd33a0075fb111b652e23feb42b60287

File: courses/free-courses-hi.md (hunk 2 above)
Comment (davorpa) [reply courses/free-courses-hi.md]: Fixed in commit 4b4b8b1bfd33a0075fb111b652e23feb42b60287

File: courses/free-courses-hi.md [hunk 7]
```diff
@@ -42,6 +47,13 @@
 * [Complete Android Development Course in Hindi](https://www.youtube.com/playlist?list=PLUhfM8afLE_Ok-0Lx2v9hfrmbxi3GgsX1) - Neat Roots
 
 
+### Bash and Shell
+
+* [shell scripting complete tutorial in hindi](https://youtube.com/playlist?list=PL9A0tISr5Ow5nZSY8-ICNAyXHJKesl_XL) - Cybersploit
+* [Bash Shell Scripting (NOOB)](https://youtube.com/playlist?list=PLzOLSdbK1deNuVMOw0EkKGSsES-rPeONe) - Noob Coders
+* [Bash Basic Commands](https://youtube.com/playlist?list=PLzOLSdbK1deOKmOiiv-o4wn7xUj6ZYzrM) - Noob Coders
```
Comment (davorpa) [inline courses/free-courses-hi.md:54]: Alphabetize `Shell scripting` section

```suggestion
* [Bash Basic Commands](https://youtube.com/playlist?list=PLzOLSdbK1deOKmOiiv-o4wn7xUj6ZYzrM) - Noob Coders
* [Bash Shell Scripting (NOOB)](https://youtube.com/playlist?list=PLzOLSdbK1deNuVMOw0EkKGSsES-rPeONe) - Noob Coders
* [shell scripting complete tutorial in hindi](https://youtube.com/playlist?list=PL9A0tISr5Ow5nZSY8-ICNAyXHJKesl_XL) - Cybersploit
```

File: courses/free-courses-hi.md [hunk 8]
```diff
@@ -99,6 +118,16 @@
...
 
 
+#### Bootstrap
+
+* [Bootstrap 3 Tutorial](https://youtube.com/playlist?list=PLjVLYmrlmjGciJ5_Ze6jDIfJpEeMYtsSe) - WsCube Tech
+* [Bootstrap Tutorial for Beginners(Hindi)](https://youtube.com/playlist?list=PLgPJX9sVy92wc38jA6JtvkA4l1xmJcKKH) - CS Geeks
+* [Bootstrap Tutorial In Hindi](https://youtube.com/playlist?list=PLdPwRNmUlk0k91-qAXTHFqMScNEuo8E5d) - CODE4EDUCATION
+* [Bootstrap Tutorial In Hindi](https://youtube.com/watch?v=vpAJ0s5S2t0) - CodeWithHarry
+* [Bootstrap Tutorial in Hindi With 2 Projects for Beginners \| Complete Bootstrap 5 Tutorial in Hindi](https://youtube.com/watch?v=QE5oQh63gGE) - Tech Gun
+* [Bootstrap tutorial in Hindi \| Best Course](https://youtube.com/playlist?list=PL7akPJI4biSIQmT7fSHWoMRaNUcRbXHFN) - CodinGyaan
```
Comment (davorpa) [inline courses/free-courses-hi.md:128]: Alphabetize `Bootstrap` section

```suggestion
* [Bootstrap tutorial in Hindi \| Best Course](https://youtube.com/playlist?list=PL7akPJI4biSIQmT7fSHWoMRaNUcRbXHFN) - CodinGyaan
* [Bootstrap Tutorial in Hindi With 2 Projects for Beginners \| Complete Bootstrap 5 Tutorial in Hindi](https://youtube.com/watch?v=QE5oQh63gGE) - Tech Gun
```

File: courses/free-courses-hi.md [hunk 9]
```diff
@@ -121,12 +150,24 @@
...
+
 #### Next.js
 
 * [Next.js Tutorials for Beginners in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9agtWvR_TZdb_r0dNI8-lDwG) - CodeWithHarry
 * [NextJS Tutorial In Hindi](https://www.youtube.com/playlist?list=PLwGdqUZWnOp2rDbpfKAeUi9f8qZMS7_cv) - Vinod Bahadur Thapa (Thapa Technical)
 
 
+#### Node.js
+
+* [NodeJS Tutorial in Hindi 2020](https://youtube.com/playlist?list=PLwGdqUZWnOp00IbeN0OtL9dmnasipZ9x8) - Thapa Technical
```
Comment (davorpa) [inline courses/free-courses-hi.md:168]:
```suggestion
* [NodeJS Tutorial in Hindi 2020](https://youtube.com/playlist?list=PLwGdqUZWnOp00IbeN0OtL9dmnasipZ9x8) - Vinod Bahadur Thapa (Thapa Technical)
```

File: courses/free-courses-hi.md [hunk 10]
```diff
@@ -121,12 +150,24 @@
 * [Web Development Course](https://www.youtube.com/playlist?list=PLfqMhTWNBTe3H6c9OGXb5_6wcc1Mca52n) - Apna College
 
 
+#### jQuery
+
+* [jQuery Tutorial for beginners in Hindi 2020](https://youtube.com/playlist?list=PL-6FWL4WVVWXmWe_HnPG0rBQmmJfGsTKS) - CSEtutorials
+* [jQuery Tutorial in Hindi](https://youtube.com/playlist?list=PLvQjNLQMdagPRDnMQPMs5-vQL_IyAB0St) - Teaching Web
+* [jQuery Tutorials in Hindi 2018](https://youtube.com/playlist?list=PLwGdqUZWnOp0X4dVwSsEd6dV49TLLCooI) - Thapa Technical
```
Comment (davorpa) [inline courses/free-courses-hi.md:157]:
```suggestion
* [jQuery Tutorials in Hindi 2018](https://youtube.com/playlist?list=PLwGdqUZWnOp0X4dVwSsEd6dV49TLLCooI) - Vinod Bahadur Thapa (Thapa Technical)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Complete instructor provided in videos

`Vinod Bahadur Thapa (Thapa Technical)`

**Review by davorpa — APPROVED** [review]
Review cut is enough for me

**Pull request merged**